src/lib/mdns/minimal/records/Txt.h

```diff
@@ -67,6 +71,7 @@ class TxtResourceRecord : public ResourceRecord
     }
 
 private:
+    friend class test::CheckOnlyServer;
```

Comment:
would it make sense to add getters instead for mEntries and count or some sort of iterator/copy out interface?

Reply:
I'm fine with either way. I figured the lack of getters was intentional, so I used a friend class to convey that the usage was unusual and just for testing. But if you're OK with getters, I'm ok with getters.

Reply:
Added getters in next commit.